Kiosk/Bid Fulfillment/ConfirmYourBidArtsyLoginViewController.swift

```diff
@@ -31,21 +30,20 @@ class ConfirmYourBidArtsyLoginViewController: UIViewController {
         bidDetailsPreviewView.bidDetails = nav.bidDetails
 
         emailTextField.text = nav.bidDetails.newUser.email ?? ""
-
-        let emailTextSignal = emailTextField.rac_textSignal()
-        let passwordTextSignal = passwordTextField.rac_textSignal()
-        RAC(nav.bidDetails.newUser, "email") <~ emailTextSignal.takeUntil(viewWillDisappearSignal())
-        RAC(nav.bidDetails.newUser, "password") <~ passwordTextSignal.takeUntil(viewWillDisappearSignal())
+        
+        let emailTextSignal = emailTextField.rac_textSignal().takeUntil(viewWillDisappearSignal())
+        let passwordTextSignal = passwordTextField.rac_textSignal().takeUntil(viewWillDisappearSignal())
+        RAC(nav.bidDetails.newUser, "email") <~ emailTextSignal
+        RAC(nav.bidDetails.newUser, "password") <~ passwordTextSignal
```

Comment:
(We talked about this in-person, summarizing here.)

The `emailTextSignal` and `passwordTextSignal` variables are used further down, but the `takeUntil` was only being applied to the first two uses of them, so I moved the call to the variable declaration.